Language/Generics/Superbounded_types/class_A03_t04.dart

```diff
@@ -8,7 +8,7 @@
 /// - `T` is an immediate subterm of a redirecting factory constructor signature
 ///
 /// @description Checks that it is a compile-time error if a super-bounded type
-/// `T` is used in a redirecting factory constructor signature
+/// is an immediate subterm of a redirecting factory constructor signature
```

Comment:
This test seems to need the same fix as Language/Generics/Superbounded_types/class_A03_t01.dart. However, it might also be redundant, is there anything new in here?

Reply:
If to make the same changes then this test will become identical to `Language/Generics/Superbounded_types/class_A03_t01.dart`. Therefore deleted.

Reply:
OK, thanks for checking!